src/nyc_trees/apps/home/templates/home/partials/training.html

```diff
@@ -1,6 +1,14 @@
+{% load utils %}
+
 {% if user.online_training_complete %}
-<h3>Training Complete!</h3>
-<p>Visit <a href="{% url 'home_page' %}">the dashboard</a> to explore the site and start mapping! You can also visit the Groups page to learn more about mapping opportunities and follow groups for event information.</p>
+<h3>Online Training Complete!</h3>
+<p>Visit <a href="{% url 'home_page' %}">the dashboard</a> to explore the site and start mapping!</p>
+
+<p>You can also visit the Groups page to learn more about mapping opportunities and follow groups for event information.</p>
```

Comment:
A link to the groups page here would follow the pattern of the other paragraphs, which link to the home and event pages.